### PR TITLE
feat(RAIN-45215): Add Hidden Azure & GCP suppressions list commands

### PR DESCRIPTION
**Pull request by rmoles**
Signed-off-by: Ross <ross.moles@lacework.net>

## Summary
Following on from https://github.com/lacework/go-sdk/pull/1069, there has been an ask to make the Azure & GCP suppressions objects available via the CLI to aide in any manual migration of the Azure & GCP suppressions

## How did you test this change?
Installed CLI locally and verified the suppression objects are returned as expected

## Issue
https://lacework.atlassian.net/browse/RAIN-45215

### CODE REVIEW

File: cli/cmd/suppressions_azure.go [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package cmd
+
+import (
+	"strings"
+
```
Comment (garyboyes) [inline cli/cmd/suppressions_azure.go:23]: Is this an extraneous new line or can it be removed?

**Review by garyboyes — APPROVED** [review]
1 comment left, but it's a nitpick, otherwise, LGTM 👍

File: cli/cmd/suppressions_azure.go [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+	var (
+		suppressions map[string]api.SuppressionV2
+		err          error
+	)
+
+	suppressions, err = cli.LwApi.V2.Suppressions.Azure.List()
+	if err != nil {
+		if strings.Contains(err.Error(), "No active Azure accounts") {
+			cli.OutputHuman("No active Azure accounts found. " +
+				"Unable to get legacy Azure suppressions")
```
Comment (dmurray-lacework) [inline cli/cmd/suppressions_azure.go:50]: Need to add a newline to the end of this, to stop outputting '% ' at the end

```suggestion
				"Unable to get legacy Azure suppressions\n")
```

File: cli/cmd/suppressions_gcp.go [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+	var (
+		suppressions map[string]api.SuppressionV2
+		err          error
+	)
+
+	suppressions, err = cli.LwApi.V2.Suppressions.Gcp.List()
+	if err != nil {
+		if strings.Contains(err.Error(), "No active GCP accounts") {
+			cli.OutputHuman("No active GCP accounts found. " +
+				"Unable to get legacy GCP suppressions")
```
Comment (dmurray-lacework) [inline cli/cmd/suppressions_gcp.go:49]:
```suggestion
				"Unable to get legacy GCP suppressions\n")
```

File: cli/cmd/suppressions_azure.go (hunk 2 above)
Comment (rmoles) [reply cli/cmd/suppressions_azure.go:50]: Nice catch

File: cli/cmd/suppressions_azure.go (hunk 1 above)
Comment (dmurray-lacework) [reply cli/cmd/suppressions_azure.go:23]: This is part of golang imports formatting. Native packages are separated from 3 party

**Review by dmurray-lacework — APPROVED** [review]

File: cli/cmd/suppressions_azure.go (hunk 1 above)
Comment (garyboyes) [reply cli/cmd/suppressions_azure.go:23]: Dead on, wasn't aware, cheers 👍

**Pull request merged**